src/fluvio.test.ts

```diff
@@ -168,7 +165,7 @@ describe('Configures a SmartModule with a Filter configuration', () => {
             JSON.parse(serverLogsFile)
         const stream = await consumer.streamWithConfig(Offset.FromBeginning(), {
             smartmoduleType: SmartModuleType.Filter,
-            smartmoduleData: wasmSmartModule.toString('base64'),
+            smartmoduleData: './fixtures/server_logs_filter.wasm',
```

Comment:
We no longer have to provide the gzipped  file here, instead we provide the path to the WASM file.

Reply:
I think that we should pass the data here instead of the path and alternative would be to rename `smartmoduleData` to `smartmoduleFile`

Reply:
Now we have `smartmoduleFile` to allow the user to provide a SmartModule WASM file w/o any gzipped compression nor base64 encoding. When a user provides the path to the `smartmoduleFile`, the NodeJS process will take care of compressing it and encoding the output as base64 to pass it internally to `smartmoduleData`.

`smartmoduleData` instead, will have the same behavior as of today, where we must pass a base64 encoded and gzipped SmartModule WASM file.

Reply:
Such changes were made here: 11fef73.
Let me know your thoughts!